client/package-lock.json

```diff
@@ -11,10 +11,12 @@
         "autoprefixer": "^10.4.19",
         "axios": "^1.7.2",
         "daisyui": "^4.12.2",
+        "framer-motion": "^12.6.3",
         "postcss": "^8.4.38",
         "react": "^18.2.0",
+        "react-daisyui": "^5.0.5",
```

Comment:
The daisyui package is already installed in the app. I am curious, does this package work differently?